### PR TITLE
feat(docs): added log level docu

### PR DESCRIPTION
**Pull request by tobiasweiss93**
Concerns CU-4wgu47

### CODE REVIEW

**Comment by meshcloud-gmbh** [comment]
Task linked: [CU-4wgu47 Enable Actuator for log-level changes during runtime](https://app.clickup.com/t/4wgu47)

**Comment by Jelledb** [comment]
Isn't this stuff more aimed at internal documentation? Or am I missing some information here

**Comment by sttomm** [comment]
> Isn't this stuff more aimed at internal documentation? Or am I missing some information here

Yeah, i also think that this should go into the Operations Handbook or Solution Handbook in Notion. @felixzieger , where is the best place to put info about how to configure logging?

File: docs/meshstack.logging.md [hunk 1]
```diff
@@ -120,6 +120,72 @@ It should be considered for the future to also log the userId, to correlate log
...
+    "loggers": {
+        "ROOT": {
+            "configuredLevel": "INFO",
+            "effectiveLevel": "INFO"
+        },
+        "com": {
+            "configuredLevel": null,
+            "effectiveLevel": "INFO"
+        },
+        "com.rabbitmq": {
```
Comment (sttomm) [inline docs/meshstack.logging.md:152]: d: Better provide an example for "io.meshcloud". And perhaps one for rabbitmq and one for http logging

File: docs/meshstack.logging.md [hunk 2]
```diff
@@ -120,6 +120,72 @@ It should be considered for the future to also log the userId, to correlate log
...
+
+```JSON
+curl --location --request POST 'http://localhost:8080/actuator/loggers/com.rabbitmq' \
+--header 'Authorization: Basic <username-password>' \
+--data-raw '{"configuredLevel": "WARN"}'
+```
+
+All actuator enpoints are secured with basic auth. Only with the basic auth token we can request the endpoints `/actuator/loggers` and `/actuator/loggers/<any-component>`.
+
+We've introduced an ansible script (`deployments/ansible/set_log_level.yml`) to change the log level for specific components. The log level can then be changed as follow: `ansible-playbook set_log_level.yml --extra-vars="log_level=debug"`.
```
Comment (sttomm) [inline docs/meshstack.logging.md:187]: d: It is not described in here how i would set the log level for i.e. "io.meshcloud.web". Please explain how to do this.

File: docs/meshstack.logging.md [hunk 3]
```diff
@@ -120,6 +120,72 @@ It should be considered for the future to also log the userId, to correlate log
...
+
+For example:
+
+```JSON
+curl --location --request POST 'http://localhost:8080/actuator/loggers/com.rabbitmq' \
+--header 'Authorization: Basic <username-password>' \
+--data-raw '{"configuredLevel": "WARN"}'
+```
+
+All actuator enpoints are secured with basic auth. Only with the basic auth token we can request the endpoints `/actuator/loggers` and `/actuator/loggers/<any-component>`.
```
Comment (sttomm) [inline docs/meshstack.logging.md:185]: d: For internal documentation also mention where the credentials can be found (vault + which basic auth user do i have to search for and in which path can i find it?)

**Review by sttomm — CHANGES_REQUESTED** [review]
proposed some small changes that should be applied when moving it to notion

**Comment by felixzieger** [comment]
> > Isn't this stuff more aimed at internal documentation? Or am I missing some information here
> 
> Yeah, i also think that this should go into the Operations Handbook or Solution Handbook in Notion. @felixzieger , where is the best place to put info about how to configure logging?

I suggest a new page under [Operations Section in meshdelivery Handbook]https://www.notion.so/meshcloud/meshDelivery-Handbook-9228ede908ce4cb4a756a1ba5c1eb282#86587d9cba6d460387a124b6fa6a6d8d).

File: docs/meshstack.logging.md (hunk 1 above)
Comment (tobiasweiss93) [reply docs/meshstack.logging.md:152]: Fixed.

File: docs/meshstack.logging.md (hunk 2 above)
Comment (tobiasweiss93) [reply docs/meshstack.logging.md:187]: Fixed

File: docs/meshstack.logging.md (hunk 3 above)
Comment (tobiasweiss93) [reply docs/meshstack.logging.md]: Extended the docs and added the vault path `meshcloud-<environment>/prometheus`

File: docs/meshstack.logging.md [hunk 4]
```diff
@@ -120,6 +120,81 @@ It should be considered for the future to also log the userId, to correlate log
 
 All log file entries always start with an UTC datetime. This is followed by the log-level (DEBUG, INFO, WARN, ERROR, ...).
 
+### Log in Meshfed
+
+We've configured our Spring Boot Applications (meshfed, kraken and identityconnector) to support log level changing during runetime. To expose this functionality we're using actuator configuration
```
Comment (sttomm) [inline docs/meshstack.logging.md:125]:
```suggestion
We've configured our Spring Boot Applications (meshfed, kraken-worker, kraken-api, replicator and identityconnector) to support log level changing during runtime. To expose this functionality we're using actuator configuration
```

File: docs/meshstack.logging.md [hunk 5]
```diff
@@ -120,6 +120,81 @@ It should be considered for the future to also log the userId, to correlate log
...
+For example:
+
+```JSON
+curl --location --request POST 'http://localhost:8080/actuator/loggers/com.rabbitmq' \
+--header 'Authorization: Basic <username-password>' \
+--data-raw '{"configuredLevel": "WARN"}'
+```
+
+All actuator enpoints are secured with basic auth. Only with the basic auth token we can request the endpoints `/actuator/loggers` and `/actuator/loggers/<any-component>`. You can find the necessary credentials in vault under `meshcloud-<environment>/prometheus`
+or you can get the creds from the vm under path `/opt/meshfed-api`. To connect to the machine you can use the anssh.sh script in `deployments` repository.
```
Comment (sttomm) [inline docs/meshstack.logging.md:194]: also mention that they are in application-local.yml

**Review by sttomm — APPROVED** [review]

File: docs/meshstack.logging.md (hunk 4 above)
Comment (tobiasweiss93) [reply docs/meshstack.logging.md:125]: Fixed

File: docs/meshstack.logging.md (hunk 5 above)
Comment (tobiasweiss93) [reply docs/meshstack.logging.md:194]: I don't think that this information is relevant for delivery.

**Comment by tobiasweiss93** [comment]
@sttomm @felixzieger I've moved the docu to https://www.notion.so/meshcloud/Logging-c9c389ed90d04816ade4329837283329 . I'll close this PR. If you have further remarks pls add comments to the notion page.

**Pull request closed without merging**